### PR TITLE
Fix remaining luminosity combinations

### PR DESCRIPTION
**Pull request by cschwan**
It seems I wasn't thorough enough in #25 and missed a few lines. I noticed that while implementing <https://github.com/NNPDF/pineappl/issues/199>.

I can't guarantee the first commit fixes everything, so I suggest we keep this open until I've tested it extensively.

### CODE REVIEW

File: src/Vlumifns_LHApdf.C [hunk 1]
```diff
@@ -335,8 +335,9 @@ double qq_12_lumi(const pdfArray& X1, const pdfArray& X2, collider c){
   if (c==pp){ return temp; }
   else if (c==ppbar) { return - temp; }
   else if(c==piso) {
+    // FIXME: here are strange, charm and bottom (anti-)quark PDFs missing
     return - ( 
-	      vuvu * (X1.u - X1.ubar)
```
Comment (cschwan) [inline src/Vlumifns_LHApdf.C:339]: This line is really worrying, the second parton doesn't have any PDFs...

Comment (cschwan) [reply src/Vlumifns_LHApdf.C:339]: Should be fixed with commit 3ab9b51253d76099a758f8c73439b28367e6acdf. I wonder what impact that bug has. This only matters for proton-isoscalar collisions.

**Comment by cschwan** [comment]
As far as it concerns me, we can now merge this. I fixed all bugs that I discovered using symmetry arguments, but this really is untested, and we should devise a way to test this (in a different Issue though).

**Comment by scarlehoff** [comment]
Ok, I'm going to merge this. We probably want to define some bunch of checks in order to make sure that everything works as expected. The problem is that I'm not sure there is any other code with isoscalar target (without hacking) we can compare against?

**Comment by cschwan** [comment]
A check that we could perform is that we
1. generate a proton-proton grid,
2. rewrite the initial state of one proton to the one of an iso-scalar, meaning that we perform the replacements u -> (u + d)/2 and d -> (u + d)/2 (also for the corresponding anti-particles), which will require support in PineAPPL and
3. compare that against the same calculation with collider = piso.

**Pull request merged**